measures/GEBAppliancesPeakPeriodShift/measure.rb

```diff
@@ -0,0 +1,230 @@
+# frozen_string_literal: true
+
+# see the URL below for information on how to write OpenStudio measures
+# http://nrel.github.io/OpenStudio-user-documentation/reference/measure_writing_guide/
+
+require 'csv'
+
+# start the measure
+class GEBAppliancesPeakPeriodShift < OpenStudio::Measure::ModelMeasure
+  # human readable name
+  def name
+    # Measure name should be the title case of the class name.
+    return 'GEBAppliancesPeakPeriodShift'
```

Comment:
Does this really only apply to appliances?

Reply:
Good point. Something like "SchedulesPeakPeriodShift" probably more appropriate.